--- tests/hypotheses/H04/test_h04_report017_source_harmonization.R.pre_order35a
+++ tests/hypotheses/H04/test_h04_report017_source_harmonization.R
@@ -457 +457 @@
-    "04088ce197cc7462de74c50b72cb0f0b5e5f34a4705c7181"
+    "04088ce197cc7462de74c50b72cb6c5b291c3b8ba4cb0f0b5e5f34a4705c7181"
@@ -886,2 +886 @@
-      fixed = TRUE,
-      ignore.case = TRUE
+      fixed = TRUE
@@ -1036 +1035 @@
-    "59afbca62bfaa069b11d0d50bc1a4bf63dc0e114c5c248c0029301917a3a08e6",
+    "59afbcd101e77ceadd7a720ef5e7bceebc22650d3e5d4b623a463d25f8bb2a4b",
@@ -1038 +1037 @@
-    "2e8bca31c0186e29174528de419d178d42672459f323c28f31e8c722c039f335"
+    "2e8bcaa35974c7aef36b1cfa9b942a4d7e1baca9cfacc2edb57365bf43aab636"
